{"topicSections":[{"identifiers":["doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/notCallable(_:)","doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/runtimeError(_:)"],"generated":true,"anchor":"Enumeration-Cases","title":"Enumeration Cases"},{"identifiers":["doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/errorDescription"],"generated":true,"anchor":"Instance-Properties","title":"Instance Properties"},{"identifiers":["doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/Error-Implementations","doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/LocalizedError-Implementations"],"generated":true,"anchor":"Default-Implementations","title":"Default Implementations"}],"sections":[],"primaryContentSections":[{"declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"InterpreterError","kind":"identifier"}]}],"kind":"declarations"}],"metadata":{"modules":[{"name":"SwiftPy"}],"role":"symbol","externalID":"s:7SwiftPy16InterpreterErrorO","roleHeading":"Enumeration","symbolKind":"enum","title":"InterpreterError","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"InterpreterError","kind":"identifier"}],"navigatorTitle":[{"text":"InterpreterError","kind":"identifier"}]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftpy\/interpretererror"]}],"identifier":{"url":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError","interfaceLanguage":"swift"},"kind":"symbol","hierarchy":{"paths":[["doc:\/\/SwiftPy\/documentation\/SwiftPy"]]},"relationshipsSections":[{"identifiers":["doc:\/\/SwiftPy\/10Foundation14LocalizedErrorP","doc:\/\/SwiftPy\/s5ErrorP","doc:\/\/SwiftPy\/s8SendableP"],"title":"Conforms To","type":"conformsTo","kind":"relationships"}],"schemaVersion":{"patch":0,"major":0,"minor":3},"references":{"doc://SwiftPy/documentation/SwiftPy/InterpreterError/errorDescription":{"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/errorDescription","abstract":[],"title":"errorDescription","url":"\/documentation\/swiftpy\/interpretererror\/errordescription","role":"symbol","type":"topic","kind":"symbol"},"doc://SwiftPy/documentation/SwiftPy/InterpreterError":{"type":"topic","abstract":[],"title":"InterpreterError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"InterpreterError"}],"role":"symbol","navigatorTitle":[{"kind":"identifier","text":"InterpreterError"}],"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError","url":"\/documentation\/swiftpy\/interpretererror","kind":"symbol"},"doc://SwiftPy/s8SendableP":{"identifier":"doc:\/\/SwiftPy\/s8SendableP","type":"unresolvable","title":"Swift.Sendable"},"doc://SwiftPy/documentation/SwiftPy/InterpreterError/Error-Implementations":{"role":"collectionGroup","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/Error-Implementations","url":"\/documentation\/swiftpy\/interpretererror\/error-implementations","abstract":[],"title":"Error Implementations","kind":"article","type":"topic"},"doc://SwiftPy/documentation/SwiftPy/InterpreterError/runtimeError(_:)":{"abstract":[],"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/runtimeError(_:)","url":"\/documentation\/swiftpy\/interpretererror\/runtimeerror(_:)","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"runtimeError","kind":"identifier"},{"text":"(","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":")","kind":"text"}],"title":"InterpreterError.runtimeError(_:)","kind":"symbol","role":"symbol","type":"topic"},"doc://SwiftPy/s5ErrorP":{"identifier":"doc:\/\/SwiftPy\/s5ErrorP","type":"unresolvable","title":"Swift.Error"},"doc://SwiftPy/documentation/SwiftPy":{"type":"topic","abstract":[{"text":"A fast and lightweight Python interpreter with Swift macro binding tools.","type":"text"}],"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy","kind":"symbol","role":"collection","url":"\/documentation\/swiftpy","title":"SwiftPy"},"doc://SwiftPy/documentation/SwiftPy/InterpreterError/notCallable(_:)":{"role":"symbol","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/notCallable(_:)","url":"\/documentation\/swiftpy\/interpretererror\/notcallable(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"notCallable"},{"kind":"text","text":"("},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":")"}],"abstract":[],"title":"InterpreterError.notCallable(_:)","kind":"symbol","type":"topic"},"doc://SwiftPy/10Foundation14LocalizedErrorP":{"identifier":"doc:\/\/SwiftPy\/10Foundation14LocalizedErrorP","type":"unresolvable","title":"Foundation.LocalizedError"},"doc://SwiftPy/documentation/SwiftPy/InterpreterError/LocalizedError-Implementations":{"role":"collectionGroup","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/LocalizedError-Implementations","url":"\/documentation\/swiftpy\/interpretererror\/localizederror-implementations","abstract":[],"title":"LocalizedError Implementations","kind":"article","type":"topic"}}}